axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/service/bankorder/BankOrderMoveServiceImpl.java

```diff
@@ -354,4 +365,28 @@ protected Currency getCurrency(BankOrderLine bankOrderLine) {
       return bankOrderLine.getBankOrder().getBankOrderCurrency();
     }
   }
+
+  protected void setPartnerBankDetails(BankOrderLine bankOrderLine, Move move) {
```

Comment:
I think this method would be more useful if it returned the bank details and had no move parameter